src/web/lib/braze/LocalMessageCache.ts

```diff
@@ -0,0 +1,76 @@
+import type appboy from '@braze/web-sdk-core';
+import { SlotName } from './BrazeMessages';
+
+const localStorageKeyBase = 'gu.brazeMessageCache';
+
+type Message = appboy.InAppMessage;
+
+interface Store {
```

Comment:
Give that this isn't specialised to our own types, is it true that this LocalStorage interface doesn't already exist somewhere in DCR or the browser platform?

Reply:
Good point, I'll check.

Reply:
There's a global `Storage` TS interface we can use, so I've switched to that.